ci/build.sh

```diff
@@ -76,6 +76,7 @@ if [ -n "$GITHUB_TOKEN" ] && [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
   echo "# This file is auto-generated." > deploy/kubernetes/fluentd-sumologic.yaml.tmpl
   sudo helm init --client-only
   sudo helm repo add falcosecurity https://falcosecurity.github.io/charts
+  sudo helm repo add fluent https://fluent.github.io/helm-charts
```

Comment:
Should we add it to documentation?

Reply:
Good catch, I will update the appropriate Docs.